libs/components/lookup/src/lib/modules/lookup/lookup-show-more-async-modal.component.html

```diff
@@ -7,7 +7,6 @@
   <sky-modal-content
     [skyViewkeeper]="['.sky-lookup-show-more-toolbar-wrapper']"
   >
-    <sky-wait [isWaiting]="isSearching"></sky-wait>
```

Comment:
was this intentionally removed?

Reply:
Nope. Great catch. I removed during testing and accidentally checked it in. Fixed.